src/tests/ftest/dfuse/fio_small.yaml

```
@@ -74,4 +74,5 @@ fio:
 dfuse:
   mount_dir: "/tmp/daos_dfuse"
   thread_count: 8
+  eq-count: 2
```

The yaml key has to match the python variable name.
```suggestion
  eq_count: 2
```
For example, for the last test run:
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10526/56/artifact/Functional%20Hardware%20Medium/dfuse/fio_small.py/job.log
The command was
```
taskset -c 0-3 /usr/bin/dfuse /tmp/daos_dfuse TestLabel_1 5BADDC41-5BCF-479E-A462-87611BF6AA9C --thread-count 8
```
Which doesn't have `--eq-count 2` in it

---

Thank you, this is fixed now and is showing up in https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10526/59/artifact/Functional%20on%20EL%208/dfuse/daos_build.py/job.log